packages/cra-template-mephisto-review/template/src/components/Pagination.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Button, Icon, Intent } from "@blueprintjs/core";
+import "./components.css";
+
+function Pagination({ page = 1, totalPages = 1, setPage = () => {} }) {
+  const MAX_CELLS = 7;
+  const CELL_MID_LEN = ~~(MAX_CELLS / 2);
```

Comment:
pretty cool! didn't know `~~` was a substitute for Math.floor for positive numbers

Reply:
Saw it on a github thread while writing this haha!

Reply:
Little things like this are cool, but should probably be commented 😉

Reply:
agreed, most folks wouldn't be able to decipher this. I'd either comment or revert to the more familiar Math.floor